### PR TITLE
Update MAPS_AND_AWAY_MISSIONS.md to latest changes

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Indicates what are the current maps in rotation and adds all the outdated, removed or not in rotation maps for record keeping.

## Why It's Good For The Game
git update

### CODE REVIEW

File: .github/MAPS_AND_AWAY_MISSIONS.md [hunk 1]
```diff
@@ -1,13 +1,26 @@
 ## MAPS
 
-/tg/station currently comes equipped with five maps.
+/tg/station currently has in rotation five maps.
```
Comment (LemonInTheDark) [inline .github/MAPS_AND_AWAY_MISSIONS.md:3]: Why change this?

File: .github/MAPS_AND_AWAY_MISSIONS.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
-
+* [AsteroidStation](https://tgstation13.org/wiki/AsteroidStation)
+* [BirdboatStation](https://tgstation13.org/wiki/BirdboatStation)
+* [CereStation](https://tgstation13.org/wiki/CereStation)
+* [DiscStation](https://tgstation13.org/wiki/Discstation)
+* [DreamStation](https://tgstation13.org/wiki/Dreamstation)
+* [EfficiencyStation](https://tgstation13.org/wiki/EfficiencyStation)
+* [MiniStation](https://tgstation13.org/wiki/MiniStation)
+* [OmegaStation](https://tgstation13.org/wiki/OmegaStation)
+* [RuntimeStation(Debugging)](https://tgstation13.org/wiki/RuntimeStation)
```
Comment (LemonInTheDark) [inline .github/MAPS_AND_AWAY_MISSIONS.md:23]: I mean, it's not in the main rotation but it's technically still in the game, this is a mite misleading

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: .github/MAPS_AND_AWAY_MISSIONS.md (hunk 2 above)
Comment (Ghilker) [reply .github/MAPS_AND_AWAY_MISSIONS.md:23]: `Outdated, removed or not in rotation maps.` why misleading?

File: .github/MAPS_AND_AWAY_MISSIONS.md (hunk 1 above)
Comment (Ghilker) [reply .github/MAPS_AND_AWAY_MISSIONS.md:3]: these are the maps in rotation, i feel that comes equipped means that these 5 are the only maps we ever had

Comment (LemonInTheDark) [reply .github/MAPS_AND_AWAY_MISSIONS.md:3]: I think that's a misreading of it, but it's not a huge deal.
You've got the syntax a bit fuzzed tho,
`/tg/station currently has in rotation five maps.` 
should prob be
`/tg/station currently has five maps in rotation.`

File: .github/MAPS_AND_AWAY_MISSIONS.md (hunk 2 above)
Comment (LemonInTheDark) [reply .github/MAPS_AND_AWAY_MISSIONS.md:23]: Cause every one of these was removed outside of runtime. Maybe give it its own section as a debug map alongside the multiz one?

Comment (Ghilker) [reply .github/MAPS_AND_AWAY_MISSIONS.md:23]: aye aye

File: .github/MAPS_AND_AWAY_MISSIONS.md [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
-
-* [BoxStation (default)](https://tgstation13.org/wiki/Boxstation)
+/tg/station currently has five maps in rotation.
 * [MetaStation](https://tgstation13.org/wiki/MetaStation)
 * [DeltaStation](https://tgstation13.org/wiki/DeltaStation)
+* [IceboxStation](https://tgstation13.org/wiki/IceboxStation)
+* [KiloStation](https://tgstation13.org/wiki/KiloStation)
+* [TramStation](https://tgstation13.org/wiki/Tramstation)
+
+Outdated, removed or not in rotation maps.
```
Comment (LemonInTheDark) [inline .github/MAPS_AND_AWAY_MISSIONS.md:10]: This can now be "Outdated or removed maps." right?

Comment (Ghilker) [reply .github/MAPS_AND_AWAY_MISSIONS.md]: done

**Pull request merged**